app/src/androidTest/java/com/github/multimatum_team/multimatum/DeadlineNotificationTest.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.multimatum_team.multimatum
```

Comment:
What's the reason behind this test? I assume it's for testing the CI.
If this is true, and if you think you're not going to be able to make tests work tonight, please delete this file.